src/pages/about.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react';
+import _ from 'lodash';
+import { Link } from 'react-router-dom';
+
+// MUI
+import {
+  Grid,
+  Card,
+  CardMedia,
+  CardContent,
+  Typography,
+} from '@material-ui/core';
+import { useStyles } from './page.style';
+
+// Context
+import { useUserListData } from '../components/profile/user/userListContext';
+
+export const AboutPage = () => {
+  const classes = useStyles();
+  const [devList, setDevList] = useState();
+  const { userList } = useUserListData();
+  useEffect(() => {
+    const userData = _.values(userList).filter(user => user?.admin === true);
```

Comment:
FYI, lodash also supports a functional programming paradigm that's pretty cool if you end up with one more level of dot chaining here:
`_(userList).values().filter(u=>u.isAdmin).value()`

Reply:
Huh. Didn't know it could be implemented like that. Changed it to reflect it.